lib/msal-core/src/Utils.ts

```diff
@@ -538,6 +538,23 @@ export class Utils {
     return url;
   }
 
+  /**
+   * @hidden
+   * @ignore
+   *
+   * Returns the anchor part(#) of the URL
+   */
+  static getHashFromUrl(urlStringOrFragment: string): string {
+    const hashIndex1 = urlStringOrFragment.indexOf("#");
+    const hashIndex2 = urlStringOrFragment.indexOf("#/");
+    if (hashIndex2 > -1) {
+      return urlStringOrFragment.substring(hashIndex2 + 2);
+    } else if (hashIndex1 > -1) {
```

Comment:
Nit pick: Don't need the `else` since you are returning in the above `if` block.

```js
if (condition1) {
  return ...;
}

if (condition2) {
  return ...;
}

return ...;
```

Reply:
Done